src/plugins/cal-events/cal-events.js

```diff
@@ -372,7 +373,27 @@ var componentName = "wb-calevt",
 	};
 
 // Bind the init event of the plugin
-$document.on( "timerpoke.wb " + initEvent, selector, init );
+//$document.on( "timerpoke.wb " + initEvent, selector, init );
+
+$document.on( "timerpoke.wb " + initEvent + " redraw" + selector +
+" close" + selector, selector, function( event ) {
+
+	var eventType = event.type,
+		$elm = $( "#" + event.target.id ),
+		calendarId = event.currentTarget.dataset.calevtSrc;
+
+	switch ( eventType ) {
+	case "timerpoke":
+	case "wb-init":
+		init( event );
+		break;
+
+	case "redraw":
```

Comment:
What is the use case behind this new event? + need to be documented in the list of events.

What is the difference with `wb-updated.wb-calevt`